circle.yml

```diff
@@ -9,14 +9,9 @@ test:
     - PATH=$PATH:~/google_appengine ./smoke_test.py
 
 deployment:
-  development:
-    branch: /(?!master)/
+  app_engine:
+    branch: /.*/
     commands:
       - rm venv
       - ~/google_appengine/appcfg.py update . -A caravel-code-reviews --version=$(git rev-parse --abbrev-ref HEAD) --oauth2_refresh_token=$OAUTH2_REFRESH_TOKEN
-
-  production:
-    branch: master
-    commands:
-      - rm venv
-      - ~/google_appengine/appcfg.py update . --version=master --oauth2_refresh_token=$OAUTH2_REFRESH_TOKEN
+      - if [ "x$(git rev-parse --abbrev-ref HEAD)" = "xmaster" ]; then ~/google_appengine/appcfg.py update . --version=$(git rev-parse --abbrev-ref HEAD) --oauth2_refresh_token=$OAUTH2_REFRESH_TOKEN; fi
```

Comment:
Ehhhh, it's a weird habit. If "git rev-parse --abbrev-ref HEAD" is empty, then without the x there'd be a syntax error. That shouldn't ever be the case, so I've disabled it.